### PR TITLE
Move to PLATFORM_TIP_DEBT

### PR DESCRIPTION
**Pull request by Betree**

### CODE REVIEW

**Comment by znarf** [comment]
Let's try to ship this and start making debts visible on the frontend (hidden by default, I already have use case for it when interacting with Hosts on support).

**Comment by Betree** [comment]
> Let's try to ship this and start making debts visible on the frontend (hidden by default, I already have use case for it when interacting with Hosts on support).

Alright, I'll prioritize next week

File: migrations/20210615074327-platform-tip-debt-kind.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  },
+
+  down: async queryInterface => {
+    await queryInterface.sequelize.query(`
+      UPDATE "Transactions"
+      SET "kind" = 'PLATFORM_TIP'
+      WHERE "kind" = 'PLATFORM_TIP_DEBT'
+    `);
+  },
+};
```
Comment (znarf) [inline migrations/20210615074327-platform-tip-debt-kind.js:20]: Where are we adding `PLATFORM_TIP_DEBT` to the `enum_Transactions_kind`?

Comment (Betree) [reply migrations/20210615074327-platform-tip-debt-kind.js:26]: It still WIP

**Review by znarf — APPROVED** [review]

**Pull request merged**